main.js

```diff
@@ -175,8 +222,9 @@ define(function (require, exports, module) {
         }
     }
 
-    function onCheckedStateChange() {
-        _preferences.set("checked", Boolean(_command.getChecked()));
+    function onCheckedStateChange(e) {
+        _preferences.set("checked", e.target._checked);
```

Comment:
This change seems odd... It seems like if there was a regression it would be due to a timing issue where command.getChecked() isn't set when the callback is invoked.  Hopefully this will go away when your [PR](https://github.com/DennisKehrig/brackets-show-whitespace/pull/32) for updating the events is merged.